ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -325,7 +326,10 @@
                    :name->label str
                    :show-row-hover? true
                    :on-select #(when (first %)
-                                 (do
+                                 (let [current-url (str/replace (str js/window.location) #"/now(.*)" "/now/")
```

Comment:
This route change escapes analytics. I'll create a work item for pushing the route change to analytics as well.